crates/sui-framework/sources/governance/genesis.move

```diff
@@ -160,12 +169,21 @@ module sui::genesis {
             storage_fund,
             genesis_chain_parameters.protocol_version,
             genesis_chain_parameters.system_state_version,
-            genesis_chain_parameters.governance_start_epoch,
             genesis_chain_parameters.chain_start_timestamp_ms,
             genesis_chain_parameters.epoch_duration_ms,
-            genesis_chain_parameters.initial_stake_subsidy_distribution_amount,
+
```

Comment:
How about we create `SystemParameters` and `StakeSubsidy` on line 164 or so before the call to `sui_system::create`, and pass these structs in, so we don't have to pass these 10 parameters through these layers of function calls?